### PR TITLE
KevS-SK1287-Update-currently-1-click-modules-errors-TRANSLATIONS

### PR DESCRIPTION
**Pull request by Kazarus4**
FR, DE and EN subs are already done. 

ES, IT, PL and PT subs in progress.

### CODE REVIEW

**Comment by Kazarus4** [comment]
I'll still check IT, PL and PT translations

**Comment by Kazarus4** [comment]
All translations are done. Need proof for DE and ENGB. this before EN duplications and merge with develop

File: pages/web_cloud/web_hosting/diagnostic_errors_module1clic/guide.pt-pt.md [hunk 1]
```diff
@@ -10,199 +10,281 @@ updated: 2023-08-21
...
 
 > [!warning]
 >
-> A responsabilidade sobre a configuração e a gestão dos serviços que a OVHcloud disponibiliza recai sobre o utilizador. Assim, deverá certificar-se de que estes funcionam corretamente.
+> A OVHcloud oferece-lhe serviços cuja configuração, gestão e responsabilidade é da sua responsabilidade. Assim, deverá assegurar o seu bom funcionamento.
 >
-> Este manual fornece as instruções necessárias para realizar as operações mais habituais. No entanto, se encontrar dificuldades, recomendamos que recorra a um [prestador de serviços especializado](https://partner.ovhcloud.com/pt/directory/) e/ou que contacte o editor do serviço. Não poderemos proporcionar-lhe assistência técnica. Para mais informações, aceda à secção [Quer saber mais?](#gofurther)?
+> Este guia fornece as instruções necessárias para realizar as operações mais habituais. No entanto, se encontrar dificuldades, recomendamos que recorra a um [fornecedor especializado](https://partner.ovhcloud.com/pt/diretory/) e/ou que contacte o editor do serviço. Não poderemos proporcionar-lhe assistência técnica. Mais informações na secção "[Quer saber mais?](#go-further)" deste guia.
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/diagnostic_errors_module1clic/guide.pt-pt.md]:
```suggestion
> Este guia fornece as instruções necessárias para realizar as operações mais habituais. No entanto, se encontrar dificuldades, recomendamos que recorra a um [fornecedor especializado](https://partner.ovhcloud.com/pt/directory/) e/ou que contacte o editor do serviço. Não poderemos proporcionar-lhe assistência técnica. Mais informações na secção "[Quer saber mais?](#go-further)" deste guia.
```

File: pages/web_cloud/web_hosting/diagnostic_errors_module1clic/guide.pt-pt.md [hunk 2]
```diff
@@ -10,199 +10,281 @@ updated: 2023-08-21
...
 
-#### Solução n°2: eliminar uma base de dados não utilizada <a name="delete-database"></a>
+Uma vez concluído, poderá instalar um novo "módulo com 1 clique".
+
+#### Solução n°3: eliminar uma base de dados não utilizada <a name="delete-the-database"></a>
 
 > [!warning]
 >
-> A operação de eliminação de uma base de dados é definitiva. Além disso, implica a supressão das salvaguardas da base em causa. Caso tenha dúvidas sobre as operações a realizar, contacte o webmaster ou um dos nossos [parceiros](https://partner.ovhcloud.com/pl/directory/).
+> A eliminação de uma base de dados é definitiva. Esta operação levará igualmente à eliminação dos backups da base de dados em causa. Em caso de dúvida, contacte o seu webmaster ou um dos nossos [parceiros](https://partner.ovhcloud.com/pt/diretory/).
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/diagnostic_errors_module1clic/guide.pt-pt.md]:
```suggestion
> A eliminação de uma base de dados é definitiva. Esta operação levará igualmente à eliminação dos backups da base de dados em causa. Em caso de dúvida, contacte o seu webmaster ou um dos nossos [parceiros](https://partner.ovhcloud.com/pt/directory/).
```

File: pages/web_cloud/web_hosting/diagnostic_errors_module1clic/guide.pt-pt.md [hunk 3]
```diff
@@ -10,199 +10,281 @@ updated: 2023-08-21
...
+Se possui um servidor [Web Cloud Databases](/pages/web_cloud/web_cloud_databases/starting_with_clouddb), pode criar uma base de dados do tamanho da sua escolha, no limite do espaço de disco alocado.
+
+Uma vez concluído, poderá instalar um novo "módulo com 1 clique".
 
 > [!primary]
 >
-> Encontre o comparativo das nossas diferentes [ofertas de bases de dados](https://www.ovhcloud.com/pt/web-hosting/options/start-sql/)
+> Neste caso, poderá guardar os dados de um único website utilizando um [script PHP ou um comando SSH](/pages/web_cloud/web_hosting/sql_database_export).
+>
+> Para qualquer questão sobre as manipulações a realizar, contacte a [comunidade OVHcloud](https://community.ovh.com/en/) ou um dos nossos [parceiros](https://partner.ovhcloud.com/pt/diretory/).<br>
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/diagnostic_errors_module1clic/guide.pt-pt.md]:
```suggestion
> Para qualquer questão sobre as manipulações a realizar, contacte a [comunidade OVHcloud](https://community.ovh.com/en/) ou um dos nossos [parceiros](https://partner.ovhcloud.com/pt/directory/).<br>
```

File: pages/web_cloud/web_hosting/diagnostic_errors_module1clic/guide.pt-pt.md [hunk 4]
```diff
@@ -10,199 +10,281 @@ updated: 2023-08-21
...
-Neste caso, [ligue-se em FTP](/pages/web_cloud/web_hosting/ftp_connection) ao seu alojamento, [salvaguarde localmente](/pages/web_cloud/web_hosting/ftp_filezilla_user_guide#transferencia-de-ficheiros) os seus dados e elimine os ficheiros que não são necessários para o funcionamento do seu site.
+Nesta situação, [ligue-se em FTP](/pages/web_cloud/web_hosting/ftp_connection) ao seu alojamento web, [efetue uma cópia de segurança localmente](/pages/web_cloud/web_hosting/ftp_filezilla_user_guide) aos seus dados e elimine os ficheiros que não são necessários para o funcionamento do seu website.
 
 > [!primary]
 >
-> Para qualquer questão relativa aos dados a eliminar para reduzir a quantidade de dados no seu alojamento, contacte a nossa [comunidade de utilizadores](https://community.ovh.com/en/) ou os [parceiros OVHcloud](https://partner.ovhcloud.com/pt/directory/).<br>
-> Não poderemos prestar-lhe assistência nesta matéria.
+> Para qualquer questão relativa à eliminação de dados para reduzir a quantidade de dados no seu alojamento web, contacte a nossa [comunidade de utilizadores](https://community.ovh.com/en/) ou os [parceiros OVHcloud](https://partner.ovhcloud.com/pt/diretory/).<br>
```
Comment (Kazarus4) [inline pages/web_cloud/web_hosting/diagnostic_errors_module1clic/guide.pt-pt.md]:
```suggestion
> Para qualquer questão relativa à eliminação de dados para reduzir a quantidade de dados no seu alojamento web, contacte a nossa [comunidade de utilizadores](https://community.ovh.com/en/) ou os [parceiros OVHcloud](https://partner.ovhcloud.com/pt/directory/).<br>
```

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**